src/types/models/carSales.ts

```diff
@@ -0,0 +1,23 @@
+import { ApiSearchListing } from "../../types/models/listing"
+import { SearchMessageLead } from "../../types/models"
+
+export interface Buyer {
+  email: string
+  firstName: string
+  lastName: string
+  phone: string
+}
+
+export type FirstListingParams = Pick<
+  ApiSearchListing,
+  "id" | "firstRegistrationDate" | "price" | "mileage"
+>
+
+export type SecondListingParams = SearchMessageLead["listing"]
```

Comment:
I would avoid extending a specific type for a view as it might change independently. The thought behind sharing was to reuse the pros of the listing we have defined. This one looks like a custom data structure for another feature (They could have used the same approach of extending the SearchListing there as well...)

Reply:
I see, so I will unify it. Thanks